### PR TITLE
Added Link of Level 0 and fixed the safari path

### PR DESCRIPTION
**Pull request by seadog007**

### CODE REVIEW

File: pages/develop.md [hunk 1]
```diff
@@ -235,7 +235,7 @@ Before you can build and install the extension, you need a developer certificate
 `git clone --recursive git@github.com:privly/privly-safari.git`
 2. Open Safari and choose "Show Extension Builder" from the Develop menu.
 3. Click the "+" button and choose "Add Extension...".
-4. Select the directory containing the safari extension (where you just cloned it to).
+4. Select the directory called "privly.safariextension" which is containing the safari extension and it is under the directory where you just cloned it to.
```
Comment (eMBee) [inline pages/develop.md]: i'd write this as:

Select the directory called "privly.safariextension" containing the safari extension inside the directory you just cloned.

Comment (smcgregor) [inline pages/develop.md]: +1

Comment (seadog007) [inline pages/develop.md]: Sorry, my English is not very good :P

**Comment by seadog007** [comment]
Fixed

**Pull request merged**